File: CTDK/121/121.cpp
#include <iostream>
using namespace std;

int main()
{
	int n;
	cout << "Nhap n: ";
	cin >> n;

	int ahh = 0;
	int att = 1;
	int at = 1;
	int i = 2;
	while (i <= n)
	{
		ahh = at + att;
		i++;
		att = at;
		at = ahh;
	}
	cout << "So hang thu " << n << " : " << ahh << endl;
	return 0;
}

File: CTDK_Python/121/_121.py
n = int(input("Nhap n: "))

if (n == 0):
    ahh = 1
elif (n == 1):
    ahh = 1
else:
    att = 1
    at = 1
    i = 2

    while (i <= n):
        ahh = at + att
        i += 1
        att = at
        at = ahh

print("Ket qua:", ahh)

